db/migrate/20161003204325_add_user_to_katello_subscription_facets.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserToKatelloSubscriptionFacets < ActiveRecord::Migration
+  def change
```

Comment:
@sean797, Thanks for reviewing!

One of the nice things about 'change' is that it will take care of removing the column, index and foreign key during the db:rollback.

Below is an example after db:migrate. Observe that they have been added.
```
katello=# \d katello_subscription_facets
                                          Table "public.katello_subscription_facets"
       Column       |            Type             |                                 Modifiers                                  
--------------------+-----------------------------+----------------------------------------------------------------------------
 id                 | integer                     | not null default nextval('katello_subscription_facets_id_seq'::regclass)
 host_id            | integer                     | not null
 uuid               | character varying(255)      | 
 last_checkin       | timestamp without time zone | 
 service_level      | character varying(255)      | 
 release_version    | character varying(255)      | 
 autoheal           | boolean                     | default false
 registered_at      | timestamp without time zone | not null default '2016-10-05 14:57:40.619863'::timestamp without time zone
 registered_through | character varying           | 
 user_id            | integer                     | 
Indexes:
    "katello_subscription_facets_pkey" PRIMARY KEY, btree (id)
    "index_katello_subscription_facets_on_user_id" UNIQUE, btree (user_id)
    "katello_subscription_facets_host_id" UNIQUE, btree (host_id)
Foreign-key constraints:
    "fk_rails_ee1659f1ac" FOREIGN KEY (user_id) REFERENCES users(id)
    "katello_subscription_facets_host_id" FOREIGN KEY (host_id) REFERENCES hosts(id)
Referenced by:
    TABLE "katello_subscription_facet_activation_keys" CONSTRAINT "katello_subscription_facet_activation_keys_sa_id" FOREIGN KEY 
(subscription_facet_id) REFERENCES katello_subscription_facets(id)
```

Below is an example after db:rollback.  Observe that they have been removed.
```
katello=# \d katello_subscription_facets
                                          Table "public.katello_subscription_facets"
       Column       |            Type             |                                 Modifiers                                  
--------------------+-----------------------------+----------------------------------------------------------------------------
 id                 | integer                     | not null default nextval('katello_subscription_facets_id_seq'::regclass)
 host_id            | integer                     | not null
 uuid               | character varying(255)      | 
 last_checkin       | timestamp without time zone | 
 service_level      | character varying(255)      | 
 release_version    | character varying(255)      | 
 autoheal           | boolean                     | default false
 registered_at      | timestamp without time zone | not null default '2016-10-05 14:57:40.619863'::timestamp without time zone
 registered_through | character varying           | 
Indexes:
    "katello_subscription_facets_pkey" PRIMARY KEY, btree (id)
    "katello_subscription_facets_host_id" UNIQUE, btree (host_id)
Foreign-key constraints:
    "katello_subscription_facets_host_id" FOREIGN KEY (host_id) REFERENCES hosts(id)
Referenced by:
    TABLE "katello_subscription_facet_activation_keys" CONSTRAINT "katello_subscription_facet_activation_keys_sa_id" FOREIGN KEY 
(subscription_facet_id) REFERENCES katello_subscription_facets(id)
```